### ISSUE TITLE
Use Let's Encrypt for managing https

### ISSUE THREAD

**Issue by sujato** [suttacentral/legacy-suttacentral#146]
Some time ago we introduced https on all our sites. This now a recommended standard for all sites. A very cool initiative in association with the Linux Foundation aims to ease and automate HTTPS:

https://letsencrypt.org/

It's still in beta, so no rush. Perhaps it's not necessary, as we have already got HTTPS. But if it becomes standard, it may make future development easier.

I heard about this from https://meta.discourse.org/t/support-for-lets-encrypt/22308

**Comment by blake-sc** [comment]
It's a good idea. I think I actually looked at this a year or so ago and it looked like a really good thing but then it was still in the pre-public-beta phase. Recently I actually tried to google the project but I couldn't find it (not that I tried very hard).

Maybe in the next couple of weeks I'll apply letsencrypt to one of the subdomains (pootle.suttacentral.net probably as) and see how that goes.
